idunn/api/places_list.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from elasticsearch import Elasticsearch
+from apistar.exceptions import BadRequest
+
+from idunn.utils.settings import Settings
+from idunn.utils.index_names import IndexNames
+from idunn.places import Place, Admin, Street, Address, POI
+from idunn.api.utils import get_geom, get_name, fetch_bbox_places, LONG, SHORT, DEFAULT_VERBOSITY_LIST, send_400
+
+from apistar import http
+
+logger = logging.getLogger(__name__)
+
+VERBOSITY_LEVELS = [LONG, SHORT]
+
+def bbox_or_400(verbosity, bbox):
+    if verbosity not in VERBOSITY_LEVELS:
+        send_400(f"verbosity {verbosity} does not belong to the set of possible verbosity values={VERBOSITY_LEVELS}")
+    if not bbox:
+        send_400(f"the bounding box cannot be None")
+
+    bbox = [float(c) for c in bbox.split(",")]
+    left, bot, right, top = bbox[0], bbox[1], bbox[2], bbox[3]
```

Comment:
the length of the list should be validated too. What if there are less or more than 4 values ?